lib/xgu/xgux.h

```diff
@@ -0,0 +1,24 @@
+#ifndef __XGUX_H
+#define __XGUX_H
+
+#include "xgu.h"
+
+inline
+void xgux_draw_arrays(XguPrimitiveType mode, unsigned int start, unsigned int count) {
+    uint32_t *p = pb_begin();
+    p = xgu_begin(p, mode);
+    p = xgu_draw_arrays(p, start, count);
+    p = xgu_end(p);
+    p = pb_end(p);
+}
+
+inline
+void xgux_set_clear_rect(uint32_t x1, uint32_t x2,
+                         uint32_t y1, uint32_t y2) {
+    uint32_t *p = pb_begin();
+    p = xgu_set_clear_rect_horizontal(p, ((x2-1)<<16)|x1);
+    p = xgu_set_clear_rect_vertical(p, ((y2-1)<<16)|y1);
```

Comment:
Shouldn't this be `(p, x1, x2-1)` and `(p, y1, y2-1)` ?
Consider naming them `left, right, top, bottom` for this one, too.